added pino-fluentd to community modules.

davidedantonio opened the pull request:
Fluentd allows you to unify data collection and consumption for a better use and understanding of data. It's able to forward your logs to:

- ElasticSearch
- MongoDB
- MySQL 
- Slack
- ...and many others

A full list of possible data output that you can use is here [https://www.fluentd.org/dataoutputs](https://www.fluentd.org/dataoutputs)

#593

davidmarkclements commented:
@davidedantonio fantastic work on the module - can you a brief note to this about the additional fluentd-elasticsearch support?

davidedantonio commented:
Sure. I would like to add a section in the README concerning the communication between fluentd and elasticsearch and how to configure it. Do you think this can be right?

davidmarkclements commented:
no I think just a couple words here, (if you like you can link to a description of that in your readme)

davidedantonio commented:
Fluentd can be configured to forward your logs to ElasticSearch (and search them with Kibana maybe?).
In order to use fluentd with ElasticSearch you need to install the fluent-plugin-elasticsearch plugin on your fluentd instance:

```
gem install fluent-plugin-elasticsearch
```

In your Fluentd configuration file, use `@type elasticsearch`. Additional configuration is optional, default values would look like this:

```
# File fluent.conf 
<source>
  @type forward
  port 24224
  bind 0.0.0.0
</source>
<match pino.*>
  @type copy
  <store>
    @type elasticsearch
    host elasticsearch
    port 9200
    flush_interval 10s
  </store>
  <store>
    @type stdout
  </store>
</match>
```

When Fluentd receives some logs from `pino-fluentd` and has flushed them to Elasticsearch after a certain interval (in this example 10 seconds), you can view, search and visualize, the log data using Kibana.

davidmarkclements commented:
sorry I think I've undercommunicated: what I mean is - can you update your PR with literally just a few words to indicate that is *also* has elasticsearch support, just in case anyone is looking at this page and wants to work with elasticsearch 

if the above isn't in pino-fluentd readme, I think it would very valuable add it

davidedantonio commented:
@davidmarkclements I'm sorry 🙏, but I did not understand what you meant. 
I've updated PR, I think now it's ok. Following your suggestion I've updated repository README file 😄

davidmarkclements commented on docs/ecosystem.md:
```diff
@@ -63,6 +63,8 @@ the logger for the [Rill framework](https://rill.site/).
 
 + [`pino-colada`](https://github.com/lrlna/pino-colada): cute ndjson formatter for pino.
++ [`pino-fluentd`](https://github.com/davidedantonio/pino-fluentd): send Pino logs to Elasticsearch, 
+MongoDB and many [others](https://www.fluentd.org/dataoutputs) through Fluentd instance.
```
```suggestion
MongoDB and many [others](https://www.fluentd.org/dataoutputs) via Fluentd.
```

davidmarkclements commented:
v5.11.1 released
